src/home.html

```diff
@@ -0,0 +1,76 @@
+                    <p>People often forget to pay attention to their food habits.</p>
+                    <p>In today's fast-paced life</p>
+                    <p>people often forget to pay attention to their food habits.</p>
+                    <p>Healthy Bites, aims to guide people toward a healthier lifestyle by providing delicious and
+                        nutritious food options</p>
+                    <p>As a well-known phrase says: "We are what we eat."</p>
+                </section>
+                <section class="foodType">
+                    <div class="food-grid">
+                        <div>
+                            <a href="/recipe">
+                                <!-- ADD IMAGE FOR BREAKFAST -->
+                                <img src="images/logoPlaceholder.webp" alt="ADD TEXT">
+                            </a>
+                            <h2>Breakfast</h2>
+                        </div>
+                        <div>
+                            <a href="foodListing/index.html?mealtype=">
+                                <!-- ADD IMAGE FOR MAIN COURSE -->
+                                <img src="images/logoPlaceholder.webp" alt="ADD TEXT">
+                            </a>
+                            <h2>Main Course</h2>
+                        </div>
+                        <div>
+                            <a href="foodListing/index.html?mealtype=">
+                                <!-- ADD IMAGE FOR DESSERT -->
+                                <img src="images/logoPlaceholder.webp" alt="ADD TEXT">
+                            </a>
+                            <h2>Dessert</h2>
+                        </div>
+                        <div>
+                            <a href="foodListing/index.html?mealtype=">
+                                <!-- ADD IMAGE FOR DRINKS -->
+                                <img src="images/logoPlaceholder.webp" alt="ADD TEXT">
+                            </a>
+                            <h2>Drinks</h2>
+                        </div>
+                    </div>
+                </section>
+                <div class="quote"></div>
+                <div class="testimonial"></div>
+                <div open class="news-letter">news-letter</div>
+            </div>
+
+            <div class="recipeContainer"></div>
+
+            <!-- Show products by type -->
+            <div class="recipeType">
+                <h2>recipe TYPE </h2>
+                <div class="renderRecipeByType">TYPE</div>
+            </div>
+
+
+            <!-- Show detiles -->
+            <div class="recipeDetails">
+                <h2>recipe DETAILS </h2>
+                <div class="renderRecipeDetails">DETAILS</div>
+            </div>
+
+
+            <!-- About -->
+            <div class="recipeAbout">
+                <h2>recipe ABOUT</h2>
+                <div class="renderAbout">ABOUT</div>
+            </div>
+
+        </main>
+
```

Comment:
@JosueBCe I ripped this out of `index.html` so that we weren't loading `index.html` into itself via the router. That just causes issues.